src/js/i18n/en-US.yml

```diff
@@ -216,7 +216,7 @@ careOptsFrontend:
             cancelBtn: Cancel
             saveBtn: Save
           timestampsView:
-            createdAt: Created
+            createdAt: Added
```

Comment:
One thing that this surfaced...  We should change all of the "Last Updated"s to just "Updated". In this case if there are any keys that use "last" we should remove that.